include/aoscene.h

```diff
@@ -2,19 +2,50 @@
 #define AOSCENE_H
 
 #include <QDebug>
+#include <QElapsedTimer>
+#include <QImageReader>
 #include <QLabel>
-#include <QMovie>
+#include <QTimer>
 
-class Courtroom;
 class AOApplication;
 
 class AOScene : public QLabel {
   Q_OBJECT
+
 public:
-  explicit AOScene(QWidget *parent, AOApplication *p_ao_app);
+  enum class image_type {
+    background,     // the background of the scene
+    foreground,     // for effects meant to appear under the character i.e.
+                    // speedlines
+    character,      // for characters
+    interjection,   // for shouts, e.g. Objection!
+    effect,         // effects, like flashes
+    fuck_interface, // for animated UI elements, i.e. chat_arrow
+  };
```

Comment:
the convention is really bad in this project for enums, but basically they should be TitleCase